.github/workflows/workflow-validation.yml

```diff
@@ -0,0 +1,29 @@
+name: Workflow Validation Testing
+
+on:
+  pull_request:
+    branches:
+      - main
+      - develop
+      - 'feature/**'
```

Comment:
We can remove the filter for `branches` as well. This is very quick and will only run for PRs that change workflows anyways 👍